guides/hosting/infrastructure/reverse-http-cache.md

```diff
@@ -226,7 +226,13 @@ sub vcl_backend_response {
 
     # Save the bereq.url so bans work efficiently
     set beresp.http.x-url = bereq.url;
-    set beresp.http.X-Cacheable = "YES";
+    
+    if(bereq.uncacheable) {
```

Comment:
@jonashrem what do you think, should we completely kill this header?
You can see in the `Age` header if i's cached or not. I think many people remove that anyway

Reply:
Being cachable and being served from cache is not necessary the same thing but yes, removing them all together or explaining how to add Debug Headers to a different section works as well. 

You could also just provide a link to https://docs.varnish-software.com/tutorials/hit-miss-logging/ as a reference for adding further Debug Headers.

Reply:
@shyim wrong annotation or wdyt ?

If second, lgtm :smile:

Reply:
@jonashrem can you adjust the PR accordingly? Thx :slightly_smiling_face: